lib/vagrant-service-manager/command.rb

```diff
@@ -6,6 +6,15 @@ module ServiceManager
     DOCKER_PATH = '/home/vagrant/.docker'
     SUPPORTED_SERVICES = ['docker', 'openshift', 'kubernetes']
     SCCLI_SERVICES = ['openshift', 'k8s']
+    KUBE_SERVICES = [
+      "etcd",
+      "kube-apiserver",
+      "kube-controller-manager",
+      "kube-scheduler",
+      "kubelet",
+      "kube-proxy",
+      "docker"
```

Comment:
@navidshaikh @budhrg As per my understanding to make sure minion works properly we should have docker service running and in our case it already running when you do restart then docker service might not be needed because we usually need a service to be restart if service is hanged or changed some configuration. I checked config file of kubernetes-node and kubernetes-client but didn't see any docker restart requirement when those config change. Again it is debatable and as per http://kubernetes.io/docs/getting-started-guides/fedora/fedora_manual_config/ guideline they also pointed `restart docker` for minion. So I would say this is not a reason to block this PR.